### PR TITLE
Test: New Name Test 2

### PR DESCRIPTION
**Pull request by wanda-carlson**
Test Test

### CODE REVIEW

**Comment by wanda-carlson** [comment]
@henry-richard Can you see if this is okay?

**Comment by henry-richard** [comment]
Sure let me see

File: src/hello.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 print("Chad's new file")
-print("Wanda")
+print("Wanda create PR")
```
Comment (henry-richard) [inline src/hello.py:2]:
```suggestion
print("Wanda create a wow PR")
```

File: src/hello.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 print("Chad's new file")
```
Comment (henry-richard) [inline src/hello.py:1]: Not anymore

**Review by henry-richard — CHANGES_REQUESTED** [review]

File: src/hello.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
-print("Chad's new file")
-print("Wanda")
+print("Chad and Wanda's new file")
```
Comment (henry-richard) [inline src/hello.py:1]: Add henry too

Comment (henry-richard) [inline src/hello.py:1]: Add Henry too

**Comment by henry-richard** [comment]
Mistake

**Review by henry-richard — APPROVED** [review]

**Pull request merged**